File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/op/delete/dao/RecordsDeleteDao.java
package ru.citeck.ecos.records3.record.op.delete.dao;

import org.jetbrains.annotations.NotNull;
import ru.citeck.ecos.records3.record.dao.RecordsDao;
import ru.citeck.ecos.records3.record.op.delete.dto.DelStatus;

import java.util.List;

public interface RecordsDeleteDao extends RecordsDao {

    List<DelStatus> delete(@NotNull List<String> records);
}

File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/op/delete/dao/RecordsDeleteDao.kt
package ru.citeck.ecos.records3.record.op.delete.dao

import ru.citeck.ecos.records3.record.dao.RecordsDao
import ru.citeck.ecos.records3.record.op.delete.dto.DelStatus

interface RecordsDeleteDao : RecordsDao {

    fun delete(records: List<String>): List<DelStatus>
}
